submissions/eve5ince/html-css-popup/css/master.css

```diff
@@ -0,0 +1,141 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	text-decoration: none;
+	list-style-type: none;
+	color: #757575;
+}
+body {
+	background: rgb(238,210,232);
+	background: linear-gradient(45deg, rgba(238,210,232,0.7889355571330094) 22%, rgba(99,252,69,0) 100%); 
+	font-family: Arial, Helvetica, sans-serif;
+}
+nav>ul {
+   display: flex;
+   justify-content: space-between;
+	margin-top: 30px;
+	margin-left: 900px;
```

Comment:
Use name of class for more informative.
Delete rule `margin-top: 30px;` and `margin-left: 900px;`
Set value `flex-end` for `justify-content`.

Reply:
Thank you kindly for reviewing!